packages/react-components/src/lib/ol-map/source/stories/mvt.stories.tsx

```diff
@@ -37,3 +38,31 @@ export const Basic = (): JSX.Element => (
     </Map>
   </div>
 );
+
+export const Styled = (): JSX.Element => (
+  <div style={mapDivStyle}>
+    <Map projection={Proj.WEB_MERCATOR}>
+      <TileLayer>
+        <TileOsm />
+      </TileLayer>
+      <VectorTileLayer
+        style={
+          new Style({
+            fill: new Fill({ color: 'red' }),
+            image: new Circle({
+              fill: new Fill({ color: 'orange' }),
+              radius: 1,
+            }),
+            stroke: new Stroke({ color: 'blue' }),
+          })
+        }
+      >
+        <MVTSource
+          options={getMVTOptions({
+            url: 'http://localhost:9090/maps/osm/{z}/{x}/{y}.pbf',
```

Comment:
should it run or just for interface compliance?
As an example use "internet" source

Reply:
i agree, but right now we don't have a free public source. when we host it ourselves we will change it accordingly.

Reply:
So it's just for compliance right now, no prob.